src/main/java/in/muthukumari/service/CustomerService.java

```diff
@@ -61,4 +63,25 @@ public static boolean addCustomer(Customer customer) throws ServiceException {
 		return isAdded;
 	}
 
+	/**
+	 * This method used to verify the login details
+	 * 
+	 * @param customer
+	 * @return
+	 * @throws ServiceException
+	 */
+	public static boolean loginUser(Customer customer) throws ServiceException {
+		boolean isVerified = false;
+		boolean isLoggedUser = false;
+		try {
+			isLoggedUser = ExistDAO.isLoginVerified(customer.getUserName(), customer.getPassword());
```

Comment:
put here itself isverified=true; why using seperate if ?

Reply:
I have changed this.